code/datums/gamemode/factions/vox_shoal.dm

```diff
@@ -223,6 +222,10 @@ var/list/potential_bonus_items = list(
 		else
 			results = "The vox raiders didn't beat the previous record of [score_to_beat]."
 
+		for (var/datum/role/R in members)
+			to_chat(R.antag.current, "<span class='notice'>The raid is over. You'll go back to the shoal in a few minutes...</span>")
+			spawn (1 MINUTES)
+				qdel(R.antag.current)
```

Comment:
This is only going to effect raiders right?

Reply:
Yes.